settings/LoanPolicyForm.js

```diff
@@ -50,7 +51,7 @@ class LoanPolicyForm extends React.Component {
   }
 
   getCurrentValues() {
-    const { stripes: { store } } = this.props;
+    const { store } = this.props.stripes;
```

Comment:
What is the reason for that change?

Reply:
I see no need for nested destructuring in this case. Unlikely some other properties of props will be used in this method, therefore, now it looks more readable.